.gitignore

```diff
@@ -42,7 +43,6 @@ android/app/libs
 *.keystore
 
 # KEYS
-keys.js
```

Comment:
Removed ignore for `keys.js`, since we haven't used it as a file since September.